src/Steam/README.md

```diff
@@ -14,7 +14,10 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
 'steam' => [
   'client_id' => null,
   'client_secret' => env('STEAM_CLIENT_SECRET'),
-  'redirect' => env('STEAM_REDIRECT_URI')
+  'redirect' => env('STEAM_REDIRECT_URI'),
+  'allowed_hosts' => [
+    'example.com',
```

Comment:
Based on https://github.com/SocialiteProviders/Providers/pull/817/files#r828544193, i guess we should at least comment this out, otherwise no one would be able to login.

Also, i guess we should move this value to ENV (comma-separated, maybe?) and make sure empty case works like a charm